sysdeps/managarm/x86_64/crt-src/Scrt1.s

```diff
@@ -0,0 +1,5 @@
+.section .text
+.global _start
+_start:
+	lea main(%rip), %rsi
```

Comment:
This needs to be `rdi` (see crt0.S, Managarm's `__mlibc_entry` takes different args than Linux' `__mlibc_entry`).